packages/components/src/components/table-stateful/shadow.tsx

```diff
@@ -485,6 +511,33 @@ export class KolTableStateful implements TableAPI {
 			this.changeCellSort(headerCell);
 		}
 	}
+	private handleSelectionChange(event: Event, value: string[]): void {
+		if (this.state._selection) {
+			const keyPropertyName = this.state._selection.keyPropertyName;
+			let selectedData: KoliBriTableDataType[] = [];
+			if (keyPropertyName) selectedData = this.state._sortedData.filter((item) => value.includes(item[keyPropertyName] as string));
```

Comment:
Das Ermitteln der selektierten Daten wird an zwei Stellen gemacht. Es wäre schön, wenn das in eine private Funktion ausgelagert und dann hier im `handleSelectionChange` und in `public async getSelection()` verwendet wird.